### DISCUSSION TITLE
Make zigpy repositories inc. ZHA Device Handlers part of Hacktoberfest 2022?

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#1061] [General]
Would you please consider making zigpy and its radio libraries, etc. repositories participate in Hacktoberfest 2022?

https://hacktoberfest.com

Hacktoberfest is a month-long annual event throughout October month open to open source software projects run by DigitalOcean in partnership with GitHub and GitLab in order to encourage people to contribute to open source.

GitHub wrote a guide blog-post for Hacktoberfest 2021 on how to participate which also applies to this year:

https://github.blog/2021-10-07-githubs-guide-hacktoberfest-2021/

That only applied to pull requests submitted during October this year and can usually yield a few extra small fixes from new contributors.

That could be a good way to incentive new developers as an introduction to making their first contribution to an open souce project.

I believe Hacktoberfest should be enabled for all zigpy libraries however it might work best for the ZHA Device Handlers repo:

https://github.com/zigpy/zha-device-handlers/

As an example see that most Home Assistant repositories have participated in Hacktoberfest for the last few years, see example:

https://github.com/home-assistant/core/labels/Hacktoberfest

https://github.com/home-assistant/frontend/labels/hacktoberfest

https://github.com/home-assistant/home-assistant.io/labels/Hacktoberfest

**Comment by dmulcahey** [comment]
The amount of spam that comes with that is not worth it IMO. We don't have enough support to do this.

**Reply by Hedda** [reply]
You do not think it would maybe be possible to ask for help with review support from other Home Assistant reviewers for Hacktoberfest during October to at least assist with the closing of any spam that comes in?
